features/admin.logs.v1/pages/logs-settings-page.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Avatar from "@oxygen-ui/react/Avatar";
+import Card from "@oxygen-ui/react/Card";
+import CardContent from "@oxygen-ui/react/CardContent";
+import Typography from "@oxygen-ui/react/Typography";
+import { DocumentIcon } from "@oxygen-ui/react-icons";
+import { AppConstants } from "@wso2is/admin.core.v1/constants/app-constants";
+import { history } from "@wso2is/admin.core.v1/helpers/history";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import {
+    GenericIcon,
+    PageLayout
+} from "@wso2is/react-components";
+import classNames from "classnames";
+import React, { FunctionComponent, ReactElement } from "react";
+import { useTranslation } from "react-i18next";
+import "./logs.scss";
+import { LogTypeCardInterface } from "../models/log-settings-models";
+import { LogType } from "../models/remote-log-publishing";
+import "./logs-settings-page.scss";
```

Comment:
It is recommended to use a separate scss file for this page.
```suggestion
import { LogTypeCardInterface } from "../models/log-settings-models";
import { LogType } from "../models/remote-log-publishing";
import "./logs.scss";
import "./logs-settings-page.scss";
```